reports/post_diagnostics.Rmd

```diff
@@ -183,14 +180,14 @@ kable(measure_decline_df, format.args = list(big.mark = ",")) %>%
 
 **How does our aggregated data compare to data from AP and the Marshall Project?** 
 
-AP and the Marshall Project report [data on COVID in prisons at the state-level](https://www.themarshallproject.org/2020/05/01/a-state-by-state-look-at-coronavirus-in-prisons) each week. Because of differences in our methodology (e.g. data sources, update frequency, etc.), we do NOT expect our statewide totals to perfectly align with data from AP/TMP. The table below highlights states where our totals differ by more than 20%. 
+AP and the Marshall Project report [data on COVID in prisons at the state-level](https://www.themarshallproject.org/2020/05/01/a-state-by-state-look-at-coronavirus-in-prisons) each week. Because of differences in our methodology (e.g. data sources, update frequency, etc.), we do NOT expect our statewide totals to perfectly align with data from AP/TMP. The table below includes states where our totals differ by more than 20% OR where TMP is reporting values higher than we are. 
 
 ``` {r, echo = F}
 # Read Marshall Project data 
 new_state_df %>% 
     filter(!is.na(UCLA) & !is.na(MP)) %>% 
-    mutate(pct_diff = scales::percent(abs((MP - UCLA) / UCLA))) %>% 
-    filter(pct_diff > 20) %>%
+    mutate(pct_diff = abs((MP - UCLA) / UCLA)) %>% 
+    filter(pct_diff > 0.2 | MP > UCLA) %>%
```

Comment:
Why `MP > UCLA`, here? It seems like if a value for MP is like 2 infections higher that should be ok and not flagged?

Reply:
Our data is always newer than MP, so our cumulative counts should really never be lower than theirs by any amount (except for when we're reporting on different populations, etc.). I sort of want to keep it since I feel like there's no downside to outputting it (especially since the table is sorted so big differences show up at the top) – but open to being overruled!

Reply:
Makes sense to me! Thanks!